src/strings/i18n.js

```diff
@@ -0,0 +1,23 @@
+import i18n from 'i18next';
+import XHR from 'i18next-xhr-backend';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import english from './english'
+
+i18n
+  .use(XHR)
+  .use(LanguageDetector)
+  .init({
+    fallbackLng: 'en',
+    debug: true,
+    // react i18next special options (optional)
+    react: {
+      wait: false,
+      bindI18n: 'languageChanged loaded',
+      bindStore: 'added removed',
+      nsMode: 'default'
+    }
+  });
+
+i18n.addResourceBundle('en', 'translations', english, true, true)
```

Comment:
Use `'translation'` instead of `'translations'` - ['translation' is the `defaultNs`](https://www.i18next.com/overview/configuration-options).   Having our bundles in the `defaultNs` would let you replace all calls of `translate('translations')` with just `translate()`.